packages/ui/core-components/src/lib/unsorted/viz/core/_Value.svelte

```diff
@@ -101,7 +115,7 @@
 		>[{placeholder}]<span class="error-msg">Placeholder: no data currently referenced.</span></span
 	>
 {:else if !error}
-	<span>
+	<span class={fontColor} style="color: {fontColor}">
```

Comment:
Why is `fontColor` being used as both a class and a style?

Reply:
thanks, originally we were trying to use tailwind styles, but then we decided to go with inline styling, will remove the class